### ISSUE TITLE
Evaluate if "safe"/"unsafe" logits give additional signal for evaluation purposes

### ISSUE THREAD

**Issue by tsunamit** [mlcommons/modeltune#44]

**Comment by tsunamit** [comment]
- log, extract, report
